src/odb/src/db/dbProperty.cpp

```diff
@@ -182,21 +162,35 @@ dbIStream& operator>>(dbIStream& stream, _dbProperty& prop)
   stream >> prop._next;
   stream >> prop._owner;
 
+  uint boolean;
+  int integer;
+  char* char_string;
+  std::string string_property;
+  double double_property;
```

Comment:
Why not put these in the relevant case (ie the smallest scope)?

Reply:
Done.